jupyterhub_singleuser_profiles/images.py

```diff
@@ -27,6 +43,37 @@ def get_default(self):
 
         return ''
 
+    def get_info(self, image_name):
+        imagestream_list = self.openshift.get_imagestreams(IMAGE_LABEL+'=true')
+        #abreviations
+        image_name, tag_name = image_name.split(':')
+        desc = 'opendatahub.io/notebook-image-desc'
+        name = 'opendatahub.io/notebook-image-name'
+        url = 'opendatahub.io/notebook-image-url'
+        software = 'opendatahub.io/notebook-software'
+        dependencies = 'opendatahub.io/notebook-python-dependencies'
+
+        if len(imagestream_list.items) == 0:
+            return None
+        else:
+            for i in imagestream_list.items:
+                if i.metadata.name == image_name:
+                    annotations = i.metadata.annotations
+                    for tag in i.spec.tags:
+                        if tag.name == tag_name:
+                            tag_annotations = tag.annotations
+                    if not tag_annotations:
```

Comment:
You need to declare `tag_annotations=None` on the same level - the way you have it now the the variable may not exisst if the `tag.name == tag_name` does not pass